### ISSUE TITLE
Empty server?

### ISSUE THREAD

**Issue by nitrodude** [wowstack/docker-classic-server#3]
Hello!

I have tried to deploy your solution on a gcloud throwaway for a couple of days now...  Map extraction of both the Windows and the Mac clients seems to pop a lot of "errors" but it seems to run through to its end...   Then when I log on to the server, there are no NPCs and no mobs...

I am using a client version 1.12.1...  Ever seen this?

**Comment by danielsreichenbach** [comment]
Hi guys, I will submit a fix for this in the next days. Am on a Business trip currently.

**Comment by danielsreichenbach** [comment]
Small update on this: originally we shipped a test database to verify things could work in a container environment like Docker or Kubernetes.

This database provides:

- world objects such as chairs, anvils, in general all usable objects not tied in to any quests
- world objects such as elevators and boats
- spirit healers and graveyards

This was done to facilitate world exploration right after the containers run. We are now working on adding a secondary database with usable world content and plan to release it this week and thus fix the issue.

Thanks for trying out our server!

**Comment by porkserver** [comment]
Any news on this database?  I am heppy to help test things.  I am running on the Mac Docker version with my OS X server can.

**Comment by AlexAdamenko** [comment]
Hi!

Is there any progress on this?

**Comment by danielsreichenbach** [comment]
@AlexAdamenko currently not. I  am too busy with work and traveling every day. I'd open to release the sources and someone else continuing the work on this though.

**Comment by oppahansi** [comment]
Release the sources please, do not let your work just sit in a corner. :)

**Comment by danielsreichenbach** [comment]
Sources incoming in [wowstack/classic-server](https://github.com/wowstack/classic-server) along with the opportunity to contribute.

**Comment by JoffreyHerard** [comment]
Got the same problem here on Docker for Mac OS X. 
No NPC, No mob, something like Firecamp ( in particular for the starter's Orc  zone ) 
Is there any progress on this?

**Comment by danielsreichenbach** [comment]
@JoffreyHerard there is an ongoing project to rebuild the database from scratch, zone by zone. I have a bunch of scripts that would converge a cmangos database into a structure usable for the server though. Not recommended but usable.

**Comment by JoffreyHerard** [comment]
@danielsreichenbach Apart from this problem, good work.
Are the scripts you're talking about available now just to try?

**Comment by danielsreichenbach** [comment]
@JoffreyHerard yes, the are available in this repo: https://github.com/wowstack/cmangos-classic-db

**Comment by yelexin** [comment]
How to install this database? I followed the guides and ran the InstallFullDB script but it caused error.
Using configuration file /opt/wowstack/etc/wowstack-world.conf.
world_server_1    | World Database total connections: 2
world_server_1    | Query executed: SELECT required_z2718_01_mangos_spell_affect FROM db_version LIMIT 1
world_server_1    | Query error: Unknown column 'required_z2718_01_mangos_spell_affect' in 'field list'
world_server_1    | Your [WORLD] database is out of date.
world_server_1    |   Installed version: `z2684_01_mangos_creature_template`
world_server_1    |   Required version:  `z2718_01_mangos_spell_affect`
world_server_1    | Apply updates following version `z2684_01_mangos_creature_template` to fix the issue.

**Comment by CedricThomas** [comment]
I have the same issue @yelexin. Did you find out the solution ?

**Comment by eugeniosegala** [comment]
I have the same issue, I'm trying to figure out how to fix it.

**Comment by kderholt** [comment]
Did any of you guys work around this @eugeniosegala @CedricThomas @yelexin , could you give us a quick update if you got time @danielsreichenbach ? tyvm

**Comment by jebbster88** [comment]
I managed to get it working by.

1. Clone this somewhere https://github.com/cmangos/mangos-classic.git
2. Copy the SQL directory somewhere accessible to the InstallDBFull.sh script that you got from https://github.com/wowstack/cmangos-classic-db
3. !IMPORTANT: In that copied sql folder, in the updates folder, delete any .sql files with a higher z number than whatever your "required version" was. In my case I deleted z2719 and higher.
4. In the InstallDBFull.config file, set the "CORE_PATH" variable to the folder containing the sql folder.
5. Run the InstallDBFull script, it should apply all the .sql files in the updates folder.

God knows if it was the right thing to do, but it seems to have worked.
